cdap-docs/admin-manual/source/operations/monitoring.rst

```diff
@@ -17,8 +17,97 @@ See the :ref:`Logging <http-restful-api-logging>`, :ref:`Metrics <http-restful-a
 and :ref:`Monitoring <http-restful-api-monitor>` HTTP APIs, and the
 :ref:`Java Client <reference:java-client-api>` for more information.
 
+
 Monitoring Utilities
 ====================
 CDAP can be monitored using external systems such as `Nagios <https://www.nagios.org/>`__; a Nagios-style plugin 
 `is available <https://github.com/caskdata/cdap-monitoring-tools/blob/develop/nagios/README.rst>`__
 for checking the status of CDAP applications, programs, and the CDAP instance itself.
+
+
+.. _monitoring-metadata-update-notifications:
+
+Metadata Update Notifications
```

Comment:
Nitin has asked that the Notification move to the Developers' Manual Building Block Metadata, and a link to it be added in the appropriate section of the Administration Manual.